sdcm/gemini_thread.py

```diff
@@ -83,7 +83,7 @@ def _generate_gemini_command(self):
         test_nodes = ",".join(self.test_cluster.get_node_cql_ips())
         oracle_nodes = ",".join(self.oracle_cluster.get_node_cql_ips()) if self.oracle_cluster else None
 
-        cmd = "{} --test-cluster={} --outfile {} --seed {} --request-timeout {}s --connect-timeout {}s".format(
+        cmd = "{} --test-cluster={} --outfile {} --seed {} --request-timeout {}s --connect-timeout {}s ".format(
```

Comment:
this space could have come before you append the "main" command with other parameters, but in any case, doesn't change us much...